server/container_exec.go

```diff
@@ -17,6 +17,22 @@ import (
 
 // Exec prepares a streaming endpoint to execute a command in the container.
 func (s *Server) Exec(ctx context.Context, req *types.ExecRequest) (*types.ExecResponse, error) {
+	c, err := s.GetContainerFromShortID(ctx, req.ContainerId)
+	if err != nil {
+		return nil, fmt.Errorf("could not find container %q: %w", req.ContainerId, err)
+	}
+
+	url, err := s.Runtime().ServeExecContainer(ctx, c, req.Cmd, req.Tty, req.Stdin, req.Stdout, req.Stderr)
+	if err != nil {
+		return nil, fmt.Errorf("could not serve exec for container %q: %w", req.ContainerId, err)
+	}
+
+	if url != "" {
+		log.Infof(ctx, "Using exec URL from conmon-rs: %s", url)
```

Comment:
theoretically it could be supported by another process, maybe saying "Using exec URL from container montior" or something?